test/e2e/tests/add-multiple-tokens.spec.js

```diff
@@ -0,0 +1,110 @@
+        await switchToNotificationWindow(driver);
+        await driver.findClickableElement({ text: 'Confirm', tag: 'button' });
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // Wait for token 1 address to populate in dapp
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+        await driver.wait(async () => {
+          const tokenAddressesElement = await driver.findElement(
+            '#tokenAddresses',
+          );
+          const tokenAddresses = await tokenAddressesElement.getText();
+          return tokenAddresses !== '';
+        }, 20000);
+
+        // Create Token 2
+        await driver.clickElement({ text: 'Create Token', tag: 'button' });
+        await switchToNotificationWindow(driver);
+        await driver.findClickableElement({ text: 'Confirm', tag: 'button' });
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // Wait for token 2 address to populate in dapp
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+        await driver.wait(async () => {
+          const tokenAddressesElement = await driver.findElement(
+            '#tokenAddresses',
+          );
+          const tokenAddresses = await tokenAddressesElement.getText();
+          return tokenAddresses.split(',').length === 2;
+        }, 10000);
+
+        // Create Token 3
+        await driver.clickElement({ text: 'Create Token', tag: 'button' });
+        await switchToNotificationWindow(driver);
+        await driver.findClickableElement({ text: 'Confirm', tag: 'button' });
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // Wait for token 3 address to populate in dapp
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+        await driver.wait(async () => {
+          const tokenAddressesElement = await driver.findElement(
+            '#tokenAddresses',
+          );
+          const tokenAddresses = await tokenAddressesElement.getText();
+          return tokenAddresses.split(',').length === 3;
+        }, 10000);
+
+        // Watch all 3 tokens
+        await driver.clickElement({
+          text: 'Add Token(s) to Wallet',
+          tag: 'button',
+        });
+
+        // Switch to watchAsset notificaiton
+        await switchToNotificationWindow(driver);
+        const multipleSuggestedtokens = await driver.findElements(
+          '.confirm-add-suggested-token__token-list-item',
+        );
+
+        // Confirm all 3 tokens are present as suggested token list
+        assert.equal(multipleSuggestedtokens.length, 3);
+        await driver.findClickableElement({ text: 'Add token', tag: 'button' });
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+
+        // Switch to fullscreen extension
+        await driver.switchToWindowWithTitle(
+          WINDOW_TITLES.ExtensionInFullScreenView,
+        );
+
+        // Check only 1 token is added bug.
```

Comment:
We should probably wait til this bug is fixed and merge the test with the expected behavior asserted?